### PR TITLE
fix: update codespaces configuration

### PR DESCRIPTION
**Pull request by aslafy-z**
- removed the unsupported `--yes` flag to the installer script
- added shell integration
- added initial dev call

(Please squash my commits)

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "postAttachCommand": "sh <(curl https://pkgx.sh) --yes && exec bash -i"
+  "postAttachCommand": "sh <(curl https://pkgx.sh) && eval $(pkgx --integrate)"
```
Comment (aslafy-z) [inline .devcontainer/devcontainer.json]:
```suggestion
  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && exec bash -i"
```

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "postAttachCommand": "sh <(curl https://pkgx.sh) --yes && exec bash -i"
+  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && bash -c 'dev; clear; echo pkgx integrated; $SHELL'"
```
Comment (aslafy-z) [inline .devcontainer/devcontainer.json]:
```suggestion
  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && exec bash -c 'dev; echo pkgx integrated; exec $SHELL'"
```

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "postAttachCommand": "sh <(curl https://pkgx.sh) --yes && exec bash -i"
+  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && exec bash -c 'dev; echo pkgx integrated and dev environment loaded; exec $SHELL'"
```
Comment (aslafy-z) [inline .devcontainer/devcontainer.json]:
```suggestion
  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && echo pkgx integrated && exec bash -c 'dev &&  and dev environment loaded || echo error loading dev environment; exec $SHELL'"
```

File: .devcontainer/devcontainer.json [hunk 4]
```diff
@@ -1,3 +1,3 @@
 {
-  "postAttachCommand": "sh <(curl https://pkgx.sh) --yes && exec bash -i"
+  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && echo pkgx integrated && exec bash -c 'dev &&  and dev environment loaded || echo error loading dev environment; exec $SHELL'"
```
Comment (aslafy-z) [inline .devcontainer/devcontainer.json]:
```suggestion
  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && clear && echo pkgx integrated && exec bash -c 'dev && echo dev environment loaded || echo error loading dev environment; exec $SHELL'"
```

File: .devcontainer/devcontainer.json [hunk 5]
```diff
@@ -1,3 +1,3 @@
 {
-  "postAttachCommand": "sh <(curl https://pkgx.sh) --yes && exec bash -i"
+  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && echo pkgx integrated && exec bash -i <(echo 'dev && echo dev environment loaded || echo error loading dev environment')"
```
Comment (aslafy-z) [inline .devcontainer/devcontainer.json]:
```suggestion
  "postAttachCommand": "sh <(curl https://pkgx.sh) && pkgx --integrate && echo pkgx integrated && exec bash -i <(echo 'dev && echo dev environment loaded || echo error loading dev environment; exec $SHELL')"
```

**Comment by jhheider** [comment]
> Please squash my commits

i'm _pretty_ religious about this. ;)

**Pull request merged**